swagger.yml

```diff
@@ -137,14 +137,45 @@ paths:
           required: false
           type: string
       responses:
-        '200':
+        "200":
           description: successful operation
           schema:
             type: array
             items:
-              $ref: '#/definitions/ScoreboardGame'
-        '400':
+              $ref: "#/definitions/ScoreboardGame"
+        "400":
           description: error
+  /player/stats:
+    get:
+      tags:
+        - games
+      summary: Individual player stats
+      description: Get a player's stats for a given week
+      operationId: getPlayerStats
+      produces:
+        - application/json
+      parameters:
+        - name: week
+          description: Week of season the game took place
+          in: query
+          required: true
+          type: integer
+          minimum: 0

```

Comment:
Don't worry about `minimum` properties. I don't even think it's supported in this version of Swagger.